content/Software and Downloads/Upload/If-your-board-is-unresponsive-stuck-resetting-or-otherwise-not-working-as-expected.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "If your board is unresponsive, stuck resetting, or otherwise not working as expected"
+---
+
+Learn what to do if your board appears to be running a sketch, but is not behaving as expected. If you're trying to upload a sketch but are getting an error in Arduino IDE, see [Errors when uploading a sketch](https://support.arduino.cc/hc/en-us/articles/4403365313810-Errors-when-uploading-a-sketch) instead.
+
+---
+
+1. Remove any connections to the RESET pin. If a black packaging foam is attached to the board, remove it.
+
+2. Check the power supply. If your board is powered by a USB cable, try using a different cable, or a different USB port on your computer. If using an external power supply, try using a USB cable.
+
+3. Check your sketch for incorrect use of [Watchdog](https://www.arduino.cc/reference/en/libraries/watchdog/) resets, or other declared reset functions, that can cause the board to reset unexpectedly. If you're able to upload your but it's not working as expected, debug your if, for and while statements with [Serial.print()](https://www.arduino.cc/reference/en/language/functions/communication/serial/print/) and check the Serial Monitor. Make sure you're not assigning values to an array outside its declared length.
+
+4. Reset the memory. Copy and paste the [BareMinimum](https://www.arduino.cc/en/Tutorial/BuiltInExamples/BareMinimum) sketch below or open it with _File > Examples > 01.Basics > BareMinimum in Arduino IDE. Then, double-tap the RESET button to put the board in bootloader mode. If it's detected on a port, upload the sketch.
+
+<iframe src=https://create.arduino.cc/example/builtin/01.Basics%5CBareMinimum/BareMinimum/preview?embed style="height:510px;width:100%;margin:10px 0" frameborder=0></iframe>
```

Comment:
I'm not able to see this iframe in  richdiff mode. Might be worth double checking if it's working.

Reply:
Yeah, it doesn't work in preview but we use it in content/Hardware/Nano Family/How-to-control-the-RGB-LED-and-Power-LED-of-the-Nano-33-BLE-boards.md so it should be fine.